packages/babel-plugin-proposal-object-rest-spread/src/index.js

```diff
@@ -37,6 +37,17 @@ export default declare((api, opts) => {
     return foundRestElement;
   }
 
+  function hasObjectPatternRestElement(path) {
+    let foundRestElement = false;
+    visitRestElements(path, restElement => {
+      if (restElement.parentPath.isObjectPattern()) {
+        foundRestElement = true;
+        path.stop();
```

Comment:
Shouldn't this be `restElement.stop()`?

Reply:
if i understand correctly, `restElement.stop()` will stop the traversing into `restElement` but continue on siblings, but `path.stop` will stop the entire `visitRestElements` traversal?

Reply:
That should be the difference between `.skip()` and `.stop()`. `path` is not generated by `visitElemenrs`: you should use it when you want to stop the traversal which provided it.

Reply:
oh. understood.

I found out that this line was copied over from https://github.com/babel/babel/blob/088d2200e1c473622d0ac7790af7be29dd8b4dc1/packages/babel-plugin-proposal-object-rest-spread/src/index.js#L31-L38

does this means that this shouldnt use `path.stop()` either?

Reply:
Yeah, that looks like an error